capstone-ui/src/Components/Feed/Feed.jsx

```diff
@@ -38,13 +48,22 @@ export default function Feed ( { loggedin }) {
       }
     }, [loggedin, user.stances, user.following]);
 
+    useEffect(() => {
+      setCurrentPage(1);
+      fetchData(1);
+    }, [searchQuery]);
+
+    useEffect(() => {
+      console.log("Total Pages:", totalPages);
+    }, [totalPages]);
```

Comment:
Why do you need this useEffect?

Reply:
This was just to check the pages with the log, I'll remove it.